packages/gasket-plugin-intl/README.md

```diff
@@ -35,9 +35,9 @@ module.exports = {
   files from (default: `locales`)
 - `outputDir` - (string) path of directory to build locale and manifest files to
   (default: `./build/locales`)
-- `assetPrefix` - (string) change the default path to `/_locale` endpoint by
+- `zone` - (string) change the default path to `/_locale` endpoint by
```

Comment:
Let's retain `assetPrefix` here in the docs:
```suggestion
- `assetPrefix` - (string) change the default path to `/_locale` endpoint by
```
Because you are not looking for `intl.zone`, but rather `intl.assetPrefix` still.

Reply:
Ah yes, great catch! Thanks!